core/src/lightclient/types.rs

```diff
@@ -184,3 +184,34 @@ impl<T: Serialize> Serialize for Rich<T> {
         }
     }
 }
+
+// TransactionReceipt
+pub struct MyTransactionReceipt(pub TransactionReceipt);
```

Comment:
I would suggest against naming this struct `MyTransactionReceipt` => might prefer
`KakarotTransactionReceipt` or `TransactionReceiptWrapper`

Reply:
That being said, it's probably simpler to create a util function, name `create_default_transaction_receipt()` that returns a `TransactionReceipt`

There is no need to create a struct here:)